hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/AccessController.java

```diff
@@ -1203,12 +1203,6 @@ public void preDecommissionRegionServers(ObserverContext<MasterCoprocessorEnviro
     requirePermission(ctx, "decommissionRegionServers", Action.ADMIN);
   }
 
-  @Override
```

Comment:
IMO we should not drop the method. If we want to allow every user to be able to be able to do so, we should have a no-op method and capture same as comments. 

Although I am curious why only this change when we have other methods around listing?

Reply:
Updated to Action.READ. However I am curious to know the difference between removing this method vs having a no-op method? 

> And also to start a discussion thread on dev list about this.

Created a thread here. https://lists.apache.org/thread/vcf50plmsx59yh4fyvsfpo7xht4rbhz8

Reply:
> However I am curious to know the difference between removing this method vs having a no-op method?

IMO removing a method may send a false impression to a future auditor that there is no Access Rule defined for the method or it is somehow missing. And the person may end up re-adding the method. So it's better to have it, even if no-op.

Reply:
Makes sense !!